### ISSUE TITLE
pimatic-cron - periodical rule

### ISSUE THREAD

**Issue by SideInfo** [pimatic/pimatic#412]
It it possible to setup a rule that is executed every i.e. hour using pimatic-cron plugin?

**Comment by incmve** [comment]
Is this implemented @sweetpi ?

**Comment by sweetpi** [comment]
No sorry accidentally closed this while playing with: https://waffle.io/pimatic/pimatic

**Comment by sweetpi** [comment]
Done :)

**Comment by incmve** [comment]
Cool! :+1:

**Comment by Yves911** [comment]
Very cool! Thanks !

**Comment by koffienl** [comment]
Can you explain how to use this in rules? Can't find how to use 'every' in the THEN section?

**Comment by Yves911** [comment]
@koffienl 
```
IF every 10 minutes THEN ...
```

**Comment by sweetpi** [comment]
I didn't think about moving it to the THEN part, but I think it would be cool to have it there, too like in:

```
IF ... THEN set $time to $time + 5 every 5 minutes
```

**Comment by leader21** [comment]
> I didn't think about moving it to the THEN part, but I think it would be cool to have it there

:+1:

**Comment by Yves911** [comment]
Yeah, didn't understood/think that it could be also good to have it in the THEN part :+1:

**Comment by mwittig** [comment]
Apparently,  it is not possible to use "IF every 1 month".

**Comment by mwittig** [comment]
Ok, sweetpi told me that every 1 month is not going to work, but `its "1st of month"` can be used as work-around

**Comment by zlazar** [comment]
So here is what i need:
If automatic mode is on then turn on a switch at every hour for 1 minute.
How could the rule looks?
This is my way, but it is a trivial solution:

"When sw_automode is on and its 8pm or when sw_ automode is on and its 9pm.....then turn x on for y minute"

Thanx

**Comment by mwittig** [comment]
@zlazar In your case the following should work

```
WHEN
sw_automode is on and every 1 hour
THEN
turn x on for 1 minute

**Comment by zlazar** [comment]
Thx, 
how can be this only betwen 8AM and 8PM?

**Comment by mwittig** [comment]
try the following:

```
... and its after 8:00 a.m. and its before 8:00 p.m.
